src/genericSchemeRegistry/index.ts

```diff
@@ -98,6 +98,11 @@ export class ActionField {
       userValue = userValue.trim();
     }
 
+    if (this.type == "uint256") {
```

Comment:
Should confirm that this doesn't break anything.

Particularly:  What if `this.decimals` is set?  It won't be executed.